`EditNumber` control added to documentation.

SetTrend opened the pull request:
The Adobe Photoshop script "Image Processor.jsx" at
`C:\Program Files\Adobe\Adobe Photoshop 2022\Presets\Scripts`
comes with an `editnumber` control for entering numerical
values.

That `editnumber` control is currently undocumented. This pull
request adds the corresponding documentation.

Klustre commented:
Hi @SetTrend thanks for the PR! Can you add a warning that this is undocumented? More info [here](https://github.com/aenhancers/javascript-tools-guide#adding-undocumented-attributes-or-methods)

zlovatt commented:
Other considerations and notes that should be added: does this work in every hostapp? How long has this been part of Extendscript?

If it's only specific to Photoshop 22+, then a warning should be added noting this.

SetTrend commented:
> Hi @SetTrend thanks for the PR! Can you add a warning that this is undocumented? More info [here](https://github.com/aenhancers/javascript-tools-guide#adding-undocumented-attributes-or-methods)

Thanks, @Klustre, for taking the time!

Where would you want me to add this warning?

Would you mind going to this PR's "Files changed" tab and click on a code line in the right code panel where you want me to add the warning? Then I can easily comply to your review comment from there.



> Other considerations and notes that should be added: does this work in every hostapp? How long has this been part of Extendscript?
> 
> If it's only specific to Photoshop 22+, then a warning should be added noting this.

@zlovatt:
This is something only an Adobe employee can answer. I'm just a user trying to help. I don't have any insights of when or where this feature was added by Adobe.

zlovatt commented:
@SetTrend This doesn't exist in core Extendscript, AE, AI, or ID. 

I don't keep other hostapps installed, so can't check those-- but so far it seems this is PS only. Can you add an [admonition](https://github.com/aenhancers/javascript-tools-guide#admonitions-usage=) detailing this, in all the places you mention editnumber?

SetTrend commented:
HI @zlovatt,

sure! Would you want to provide me with the type of admonition to use? What would you suggest should the admonition's text be to fit your requirements?

zlovatt commented:
@SetTrend See the ["Admonitions Usage" section of the readme](https://github.com/aenhancers/javascript-tools-guide#admonitions-usage) for how to use admonitions.

In this case, a "Note" would be consistent with other docs; something like this:
```rst
.. note::
	This functionality was added in Photoshop XX.X (CC XX).
```

SetTrend committed:
`EditNumber` control added to documentation.

The Adobe Photoshop script "Image Processor.jsx" at
C:\Program Files\Adobe\Adobe Photoshop 2022\Presets\Scripts
comes with an `editnumber` control for entering numerical
values.

This `editnumber` control is currently undocumented. My pull
request adds the corresponding documentation.

SetTrend commented:
Done. I've now added notes to all places where `EditNumber` is mentioned.

SetTrend commented:
Any news on this?

zlovatt commented:
I know I gave you the boilerplate message there, but could you update it so it reflects that it was _only_ added to Photoshop, and that it doesn't exist anywhere else? Went to approve and felt that it'd be misleading to not make that super explicit. Sorry!
